shared/js/dataDriver.js

```diff
@@ -343,11 +359,17 @@ loop.DataDriver = function() {
       // Determine the clock skew before continuing initialization.
       this.update("meta", "lastConnect").then(({ timestamp }) => {
```

Comment:
I'd suggest that the arrow function move into it's own top-level function, since it actually does something other than just connecting to the room.  I would expect it to also simplify your unit tests, since there would be two simple units in play rather than one more complex one.

If you feel strongly that this won't be the worth the work, you can skip it.  

That said, in either case, it'd be great if _connectToRoom were _connectToRoomAndUpdate or something similar so that folks reading the calling code understand the implied side-effects to some degree.